### PR TITLE
[FMT] apply black nilearn.plotting.display

### PR DESCRIPTION
**Pull request by Remi-Gau**
Relates to #2528 

Changes proposed in this pull request:

- update configs to include nilearn/plotting/display
- apply black

### CODE REVIEW

File: nilearn/plotting/displays/_figures.py [hunk 1]
```diff
@@ -62,11 +64,13 @@ def __init__(self, figure=None, output_file=None):
         try:
             import plotly.graph_objects as go
         except ImportError:
-            raise ImportError("Plotly is required to use "
-                              "`PlotlySurfaceFigure`.")
+            raise ImportError(
+                "Plotly is required to use " "`PlotlySurfaceFigure`."
```
Comment (ymzayek) [inline nilearn/plotting/displays/_figures.py:68]:
```suggestion
                "Plotly is required to use `PlotlySurfaceFigure`."
```

File: nilearn/plotting/displays/_figures.py [hunk 2]
```diff
@@ -62,11 +64,13 @@ def __init__(self, figure=None, output_file=None):
...
+            )
         if figure is not None and not isinstance(figure, go.Figure):
-            raise TypeError("`PlotlySurfaceFigure` accepts only "
-                            "plotly figure objects.")
+            raise TypeError(
+                "`PlotlySurfaceFigure` accepts only " "plotly figure objects."
```
Comment (ymzayek) [inline nilearn/plotting/displays/_figures.py:72]:
```suggestion
                "`PlotlySurfaceFigure` accepts only plotly figure objects."
```

File: nilearn/plotting/displays/_figures.py [hunk 3]
```diff
@@ -93,8 +97,9 @@ def savefig(self, output_file=None):
         try:
             import kaleido  # noqa: F401
         except ImportError:
-            raise ImportError("`kaleido` is required to save plotly "
-                              "figures to disk.")
+            raise ImportError(
+                "`kaleido` is required to save plotly " "figures to disk."
```
Comment (ymzayek) [inline nilearn/plotting/displays/_figures.py:101]:
```suggestion
                "`kaleido` is required to save plotly figures to disk."
```

File: nilearn/plotting/displays/_slicers.py [hunk 4]
```diff
@@ -136,31 +153,40 @@ def init_with_figure(cls, img, threshold=None,
...
             if leave_space:
                 figsize[0] += 3.4
-            figure = plt.figure(figure, figsize=figsize,
-                                facecolor=facecolor)
+            figure = plt.figure(figure, figsize=figsize, facecolor=facecolor)
         if isinstance(axes, plt.Axes):
-            assert axes.figure is figure, ("The axes passed are not "
-                                           "in the figure")
+            assert axes.figure is figure, (
+                "The axes passed are not " "in the figure"
```
Comment (ymzayek) [inline nilearn/plotting/displays/_slicers.py:165]:
```suggestion
                "The axes passed are not in the figure"
```

File: nilearn/plotting/displays/_slicers.py [hunk 5]
```diff
@@ -260,8 +299,9 @@ def add_overlay(self, img, threshold=1e-6, colorbar=False,
             is computed based on the data.
         """
         if colorbar and self._colorbar:
-            raise ValueError("This figure already has an overlay with a "
-                             "colorbar.")
+            raise ValueError(
+                "This figure already has an overlay with a " "colorbar."
```
Comment (ymzayek) [inline nilearn/plotting/displays/_slicers.py:303]:
```suggestion
                "This figure already has an overlay with a colorbar."
```

**Review by ymzayek — COMMENTED** [review]
Otherwise, LGTM!

File: nilearn/plotting/displays/_slicers.py [hunk 6]
```diff
@@ -455,17 +514,19 @@ def _show_colorbar(self, cmap, norm, cbar_vmin=None,
...
+            x1 - (x_adjusted_width + x_adjusted_margin),
+            y0 + self._colorbar_margin["top"],
+            x_adjusted_width,
+            height
+            - (self._colorbar_margin["top"] + self._colorbar_margin["bottom"]),
+        ]
         self._colorbar_ax = figure.add_axes(lt_wid_top_ht)
-        if _compare_version(matplotlib.__version__, '>=', "1.6"):
-            self._colorbar_ax.set_facecolor('w')
+        if _compare_version(matplotlib.__version__, ">=", "1.6"):
```
Comment (ymzayek) [inline nilearn/plotting/displays/_slicers.py:526]: Note to self: check where `_compare_version` is used in codebase referencing package versions we don't support anymore and remove extraneous code

Will open an issue

**Review by bthirion — APPROVED** [review]
LGTM, thx.

**Pull request merged**